Makefile

```diff
@@ -333,6 +335,9 @@ build-windows: .prebuild prepare-enterprise
 
 build: build-linux build-windows build-osx
 
+build-current: .prebuild prepare-enterprise
```

Comment:
It's the most common use case when you're developing and you know what you're doing, but I'm thinking of people who don't read from documentation or just want to build from source without modifications. Typically running `make` doesn't start up docker containers and a web server in the background; building an executable for the current platform is closer to what it would usually do